web/src/lib/components/assets/thumbnail/thumbnail.svelte

```diff
@@ -85,11 +88,11 @@
 </script>
 
 <IntersectionObserver once={false} on:intersected let:intersecting>
-  <!-- svelte-ignore a11y-no-noninteractive-tabindex -->
-  <div
+  <a
+    href={isMultiSelectState ? undefined : `${currentUrlReplaceAssetId(asset.id)}`}
```

Comment:
Is the tertiary needed? Can the href attribute always be set to `currentUrlReplaceAssetId(asset.id)` since the click event handler will now prevent the default when isMultiSelectState is true?

Reply:
You're correct. I've just pushed a change to correct this. Thanks!